README.md

```diff
@@ -64,6 +62,8 @@ nano /etc/network/interfaces
 auto lo
 iface lo inet loopback
 
+iface eth0 inet dhcp
```

Comment:
we can consider to add 
```
auto eth0
allow-hotplug eth0
```
I believe I needed that for getting the ethernet to connect when I plugged it in to my mac which was sharing etherent, 
but thats really separate from this PR 
just noting it as you added this line